static/locale/bg/LC_MESSAGES/django.po

```diff
@@ -1067,10 +1067,6 @@ msgstr ""
 msgid "All streams"
 msgstr "Всички канали"
 
-#: templates/zerver/left_sidebar.html:78
-msgid "SHARE THE LOVE"
-msgstr ""
-
```

Comment:
As a general rule, you don't need to edit translation files; they're automatically generated.